### PR TITLE
Vue.html updated!

### PR DESCRIPTION
**Pull request by haiderali171**
@LeeWannacott Hi there, after a long studious fight to find the solution I was able to come up with something. During my research, it was the most common code that can be used for your library in vue. It is simple and easy because I have tested it. Other solutions can be complicated. You told me to avoid them! I will try to find a way to make other solutions simpler. But now, if you accept my pull request.. I will start work on the Angular examples and then later come onto other Vue examples!

Regards!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,9 @@
     "react": "^17.0.0",
     "react-dom": "^17.0.0",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "^4.0.3"
+    "react-scripts": "^4.0.3",
+    "table-sort-js": "*",
```
Comment (strager) [inline package.json:14]: How can table-sort-js depend upon itself?

Comment (haiderali171) [reply package.json]: I had installed table-sort-js globally while testing the actual code somewhere else. Perhaps, that is why it was depending on itself. Have made the neccessary changes!

**Comment by haiderali171** [comment]
@LeeWannacott Please see my previous comment and I think I have resolved the issue because the CI Tests are not showing any build failures! I had installed table-sort-js globally while testing the actual code somewhere else. Perhaps, that is why it was depending on itself. Have made the neccessary changes..

File: public/docs/vue.html [hunk 2]
```diff
@@ -60,9 +62,98 @@ <h2>table-sort-js</h2>
...
+            &lt;title>Table-Sort-JS&lt;/title&gt;
+          &lt;/head&gt;
+
+          &lt;body&gt;
+            &lt;div id="app"&gt;&lt;/div&gt;
+          &lt;/body&gt;
+
+          &lt;/html&gt;
+
+</code></pre>
```
Comment (LeeWannacott) [inline public/docs/vue.html:112]: Not sure if we need to say this for HTML :thinking: , it's not really unique to table-sort-js. index.html has `<div id ="app">` by default anyway; so they don't need to insert the markup into their HTML.

**Review by LeeWannacott — CHANGES_REQUESTED** [review]
Seems to work :100:. Just some minor stuff to consider. This example is for Vue 2.0; we should make an example for Vue 3.0 as well, if you are interested?; considering it just came out at the end of last year. But if you fix up the minor stuff I'm happy to merge this. :+1:

**Comment by haiderali171** [comment]
> Seems to work 💯. Just some minor stuff to consider. This example is for Vue 2.0; we should make an example for Vue 3.0 as well, if you are interested?; considering it just came out at the end of last year. But if you fix up the minor stuff I'm happy to merge this. 👍

Can I look into the Vue 3.0 example after the Angular examples? I'm keen on working the Angular examples now! What say?

**Comment by LeeWannacott** [comment]
@haiderali171 .I may have made a mistake and force pushed without fetching facepalm you wouldn't mind redoing the PR without package.json or package-lock.json to make it easier?

**Pull request merged**